README.md

```diff
@@ -38,21 +38,32 @@ spark-submit \
 Datasets Comparison 
 Usage: spark-submit [spark options] --class za.co.absa.hermes.datasetComparison.DatasetComparisonJob hermes.jar [options]
 
-  -f, --raw-format <value> format of the raw data (csv, xml, parquet,fixed-width, etc.)
-  --rowTag <value>        use the specific row tag instead of 'ROW' for XML format
-  --delimiter <value>      use the specific delimiter instead of ',' for CSV format
-  --header <value>         use the header option to consider CSV header
+  -f, --format <value>     format of the raw data (csv, xml, parquet,fixed-width, etc.).
```

Comment:
This is a printout of former `--help` I am not sure all the info here is correct in general. For example, there are no defaults enforced by us anymore. The defaults are spark defaults, so I would not speak about them in the README and only tell the user to follow spark defaults.